src/utils.js

```diff
@@ -238,6 +240,22 @@ function urlValidation(urlString) {
 
 const normalizeFileUpload = (value) => value && value.split(/\r\n|\n/);
 
+function convertChannelCodeToDisplayName(code) {
+  if (code === 'SAP') {
+    return code;
+  }
+  if (code === 'DEGREED2') {
+    return 'Degreed';
+  }
+  if (code === 'DEGREED') {
+    return 'Degreed_old';
+  }
+  if (code === 'CSOD') {
```

Comment:
Optional: If you are up for it, I think we have an opportunity to have a nice map (dictionary) where keys are channel name and values are objects containing all the needed info like Icon (used above), displayName (here) etc. But only do it if you feel it will improve the cleanliness, it's just a tad bit cleaner maybe than using several functions with switches one for each mapping